### PR TITLE
chore: add bug report template

### PR DESCRIPTION
**Pull request by miparnisari**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  type organization
+    relations
+      define member: [user]
+tuples:
+  - user: user:anne
+    relation: member
+    object: organization:openfga
+  - user: user:bob
+    relation: member
+    object: organization:openfga
```
Comment (aaguiarz) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]: Can we also provide as an example check/list-tuples tests, so they can demonstrate how the behavior is not what they expect?

Comment (miparnisari) [reply .github/ISSUE_TEMPLATE/bug_report.md:36]: My hope was that they put that in the summary section. But i'll add it here.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+tests: # remove this if not a bug in Check or ListObjects API
+  - name: test-1
+    check:
+      - user: user:anne
+        object: organization:openfga
+        assertions:
+          member: true
+    list-objects:
```
Comment (rhamzeh) [inline .github/ISSUE_TEMPLATE/bug_report.md:44]:
```suggestion
    list_objects:
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    relation: member
+    object: organization:openfga
+  - user: user:bob
+    relation: member
+    object: organization:openfga
+```
+
+**Other data**
+- How are you running OpenFGA? (As a binary, in Docker, in Kubernetes)
+- What datastore are you using? (In memory, MySQL, Postgres)
```
[the remark this answers is not in the document]
Comment (miparnisari) [reply .github/ISSUE_TEMPLATE/bug_report.md:55]: That's the question below :)

[the remark this answers is not in the document]
Comment (rhamzeh) [reply .github/ISSUE_TEMPLATE/bug_report.md:55]: Can we phrase it a bit differently to ensure they understand it also applies for config?

```
- Are you running OpenFGA with any configuration overrides or with any of the flags mentioned in `./openfga run --help`?
```

**Comment by matthewpereira** [comment]
Is it worth adding a heading or a note to prompt them for **Steps to Reproduce**? In my experience this also helps people differentiate between real bugs and questions/ideas, but I'm open to feedback from other people's experiences with bug reporting workflows.

**Review by rhamzeh — APPROVED** [review]

**Pull request merged**